proto/spacemesh/v1/smesher_types.proto

```diff
@@ -77,49 +69,42 @@ enum ComputeApiClass {
     COMPUTE_API_CLASS_VULKAN = 3;
 }
 
-// Basic post data. Used for users to provide requested post data and by
-// other messages which contain post data.
-message PostData {
-    string path = 1; // User provided path to create the post data files at
-    uint64 data_size = 2; // Requested post data size
-    bool   append = 3; // Append to existing files if they exist. Otherwise overwrite.
-    bool   throttle = 4; // Throttle down setup phase computations while user is interactive on system
-    uint32 provider_id = 5; // A PostProvider id
+// Post init options. Used to define the requested options and by
+// other messages which contain the options.
+message PostInitOpts {
+    string data_dir = 1; // User provided path to create the post data files at
+    uint32 num_units = 2; // Number of PoST data commitment units to generate
+    uint32 num_files = 3; // Number of files to equally distribute the labels among
+    uint32 compute_provider_id = 4; // A PostProvider id
+    bool throttle = 5; // Throttle down setup phase computations while user is interactive on system
 }
 
 message CreatePostDataRequest {
-    PostData data = 1;
+    PostInitOpts opts = 1;
 }
 
 // Param passed to methods to indicate a request to delete data files
 message StopSmeshingRequest {
     bool delete_files = 1;
 }
 
-message StopPostDataCreationSessionRequest {
-    bool delete_files = 1;
+message PostComputeProvidersRequest {
+    bool benchmark = 1;
```

Comment:
What does benchmark mean here?

Reply:
Request to run a short benchmarking session for each provider, so that the `performance` field of `PoSTSetupComputeProvider` will be assigned with the num of hashes per sec.

Documentation is still missing and will be complete soon.